core/src/main/java/org/wso2/msf4j/internal/router/HttpMethodInfo.java

```diff
@@ -16,12 +16,16 @@
 
 package org.wso2.msf4j.internal.router;
 
+import javafx.util.Pair;
```

Comment:
javafx.util.Pair is removed. org.wso2.msf4j.internal.router.ImmutablePair is used instead